addons/website/static/src/js/editor/widget_link.js

```diff
@@ -38,39 +43,81 @@ weWidgets.LinkTools.include({
             classes: {
                 "ui-autocomplete": 'o_website_ui_autocomplete'
             },
-        }
-        wUtils.autocompleteWithPages(this, this.$('input[name="url"]'), options);
+        };
+        this.$urlInput = this.$('input[name="url"]');
+        $('.o_we_link_action_change').toggleClass('d-none', !this.$urlInput.val().includes('/web/'));
+        wUtils.autocompleteWithPages(this, this.$urlInput, options);
         this._adaptPageAnchor();
         return def;
     },
+    isDownloadEnable: function () {
+        return this.$link.attr('data-link-action') === 'download';
```

Comment:
Do not use jquery anymore and use `.dataset`

Reply:
Could you explain to me why it's better do to so ? Since `$link` is already a jquery element,  it seems counterintuitive to unpack it then access it's dataset (`this.$link[0].dataset.linkAction`). Thanks in advance ! :)

Reply:
Because we need to start somewhere to get rid of jQuery step by step :wink: